tests/security/cc/cc_audit_test_setup.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Setup "audit-test" test environment of a system with needed packages
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93441
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use registration 'add_suseconnect_product';
+
+sub run {
+    my ($self)   = @_;
+    my $dir      = $audit_test::testdir;
+    my $file_tar = $audit_test::testfile_tar . ".tar";
+
+    select_console "root-console";
+
+    # Add needed modules
+    add_suseconnect_product('sle-module-legacy');
+    add_suseconnect_product('sle-module-desktop-applications');
```

Comment:
Lili, desktop is not a must one IMO, please double check :)

Reply:
> Lili, desktop is not a must one IMO, please double check :)

I do not know atm. but I follow the docs offered by cert team. Let's keep it atm in case some test cases are needed as we are not sure.

Reply:
Also, we do not run all the audit-test cases as good as baseline (time consuming tasks).